cglims/cli/commands.py

```diff
@@ -77,19 +77,22 @@ def config(context, gene_panel, family_id, samples, capture_kit, customer_or_cas
 @click.command()
 @click.option('-c', '--condense', is_flag=True, help='condense output')
 @click.option('-p', '--project', is_flag=True, help='identifier is a project')
+@click.option('-n', '--name', is_flag=True, help='identifier is the customer sample name')
```

Comment:
Sure. I only took `name` as this is the name of the XML element that holds the external sample name. I'll rename.